src/main/java/com/foo/gosucatcher/domain/member/presentation/MemberController.java

```diff
@@ -0,0 +1,140 @@
+package com.foo.gosucatcher.domain.member.presentation;
+
+import java.net.MalformedURLException;
+import java.nio.charset.StandardCharsets;
+
+import javax.validation.constraints.Email;
+
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.UrlResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.util.UriUtils;
+
+import com.foo.gosucatcher.domain.member.application.MemberService;
+import com.foo.gosucatcher.domain.member.application.dto.request.MemberInfoChangeRequest;
+import com.foo.gosucatcher.domain.member.application.dto.request.MemberLogInRequest;
+import com.foo.gosucatcher.domain.member.application.dto.request.MemberSignUpRequest;
+import com.foo.gosucatcher.domain.member.application.dto.request.ProfileImageUploadRequest;
+import com.foo.gosucatcher.domain.member.application.dto.response.MemberLogInResponse;
+import com.foo.gosucatcher.domain.member.application.dto.response.MemberPasswordFoundResponse;
+import com.foo.gosucatcher.domain.member.application.dto.response.ProfileImageUploadResponse;
+import com.foo.gosucatcher.domain.member.domain.ImageFile;
+import com.foo.gosucatcher.global.error.ErrorCode;
+import com.foo.gosucatcher.global.error.exception.InvalidValueException;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v1/members")
+public class MemberController {
+
+	private final MemberService memberService;
+
+	public MemberController(MemberService memberService) {
+		this.memberService = memberService;
+	}
+
+	@PostMapping("/signup")
+	public ResponseEntity<Void> signUp(
+		@RequestBody @Validated MemberSignUpRequest memberSignUpRequest) {
+		memberService.signUp(memberSignUpRequest);
+
+		return new ResponseEntity<>(HttpStatus.CREATED);
+	}
+
+	@PostMapping("/login")
+	public ResponseEntity<MemberLogInResponse> logIn(
+		@RequestBody @Validated MemberLogInRequest memberLogInRequest) {
+		var response = memberService.logIn(memberLogInRequest);
+
+		return ResponseEntity.status(HttpStatus.OK)
+			.body(response);
```

Comment:
```suggestion
		return ResponseEntity.ok(response);
```
더욱 간편하게 한줄로도 처리 가능합니다!